projects/bp-gallery/src/components/views/exhibitions/ExhibitionOverview.tsx

```diff
@@ -0,0 +1,398 @@
+import {
+  Box,
+  Button,
+  Card,
+  CardActionArea,
+  CardContent,
+  CardMedia,
+  IconButton,
+  Modal,
+} from '@mui/material';
+import {
+  useCreateExhibitionMutation,
+  useDeleteExhibitionMutation,
+  useGetExhibitionsQuery,
+} from '../../../graphql/APIConnector';
+import { useSimplifiedQueryResponseData } from '../../../graphql/queryUtils';
+import { FlatExhibition } from '../../../types/additionalFlatTypes';
+import { AuthRole, useAuth } from '../../provider/AuthProvider';
+import { useVisit } from '../../../helpers/history';
+import { useTranslation } from 'react-i18next';
+import { Dispatch, SetStateAction, useCallback, useContext, useState } from 'react';
+import { Delete } from '@mui/icons-material';
+import { t } from 'i18next';
+import { asUploadPath } from '../../../helpers/app-helpers';
+import RichText from '../../common/RichText';
+import { MobileContext } from '../../provider/MobileProvider';
+
+const ExhibitionCard = ({
```

Comment:
"Warning: validateDOMNesting(...): <button> cannot appear as a descendant of <button>."